Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+
+  <PropertyGroup>
+    <OtherFlags>--warnon:1182 $(OtherFlags)</OtherFlags>
```

Comment:
@gbaydin This turns on the warning for all projects in the repository.  You can put other build settings in this file and they will also apply across the whole repository, it;s a new .NET SDK thing